projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -86,112 +36,52 @@
   &.header-1 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-bg-color,
-      header-variables.$clr-header-bgColor,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-bg-color;
   }
 
   &.header-2 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-2-bg-color,
-      header-variables.$clr-header-2-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-2-bg-color;
   }
 
   &.header-3 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-3-bg-color,
-      header-variables.$clr-header-3-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-3-bg-color;
   }
 
   &.header-4 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-4-bg-color,
-      header-variables.$clr-header-4-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-4-bg-color;
   }
 
   &.header-5 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-5-bg-color,
-      header-variables.$clr-header-5-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-5-bg-color;
   }
 
   &.header-6 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-6-bg-color,
-      header-variables.$clr-header-6-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-6-bg-color;
   }
 
   &.header-7 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-7-bg-color,
-      header-variables.$clr-header-7-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-7-bg-color;
   }
 
   // dark theme only header added by external team to workaround lack of header-7 in dark theme
   &.header-8 {
-    @include mixins.css-var(
-      background-color,
-      clr-header-8-bg-color,
-      header-variables.$clr-header-8-bg-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: header-variables.$clr-header-8-bg-color;
   }
 }
 
 @mixin header-branding() {
   .branding {
     display: flex;
     flex: 0 0 auto;
-    @include mixins.css-var(
-      min-width,
-      clr-header-branding-min-width,
-      variables.$clr_baselineRem_8_5,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      padding,
-      clr-header-branding-padding,
-      0 variables.$clr_baselineRem_1,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      height,
-      clr-header-height,
-      header-variables.$clr-header-height,
-      variables.$clr-use-custom-properties
-    );
+    min-width: header-variables.$clr-header-branding-min-width;
+    padding: header-variables.$clr-header-branding-padding;
```

Comment:
I don't think globals belong in the main component file. Currently I accept it as preference and personal decision, so I am not bringing it up on the reviews I make. 
Otherwise we should have a discussion about it.